### PR TITLE
Etherpad OpenShift Template example

### PR DESCRIPTION
**Pull request by djixyacine**
* This serves as an OpenShift template example, which automatically
deploys Etherpad with a MongoDB backend and persistant storage.
The deployments can be tweaked thanks to the template variables.

* Write the documentation explaining the template usage from the
command line and from the web user interface.

* The Etherpad docker image which has been used was built with the
Dockerfile under `container-src` along with the entrypoint script
and the settings file.

### CODE REVIEW

File: examples/etherpad/container-src/settings.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+"ip": "0.0.0.0",
+"port" : 9001,
+
+"showSettingsInAdminPage" : true,
+
+"dbType" : "mongodb",
+"dbSettings" : {
+                 "user"    : "DB_USERNAME",
+                 "host"    : "DB_IP",
+                 "port"    : DB_PORT,
```
Comment (rlaurika) [inline examples/etherpad/container-src/settings.json:15]: Quotes could be added around "DB_PORT".

**Review by rlaurika — CHANGES_REQUESTED** [review]
I had a quick look but did not yet have time for a full review. Looks good!

Some comments based on a quick review:

- It looks like a .DS_Store file snuck in to the commit - that one could be removed
- The branch could use a rebase to master
- I'm not sure what the arrow between the service object signifies. I guess there should be an arrow from the Etherpad pod to the MongoDB service?

I think this is good for a demo. I tested this on OpenShift already and it worked well. The instructions were clear and good.

File: examples/etherpad/container-src/settings.json (hunk 1 above)
Comment (djixyacine) [reply examples/etherpad/container-src/settings.json:15]: As this is the settings file for Etherpad, and Etherpad is picky about the database port parameter being an integer (not a string), adding the quotes would make it crash.

**Review by rlaurika — APPROVED** [review]

**Comment by rlaurika** [comment]
Looks good to me!

**Pull request merged**